aidial_sdk/chat_completion/request.py

```diff
@@ -205,6 +205,7 @@ class ChatCompletionRequest(AzureChatCompletionRequest):
     addons: Optional[List[Addon]] = None
     max_prompt_tokens: Optional[PositiveInt] = None
     custom_fields: Optional[ChatCompletionRequestCustomFields] = None
+    custom_application_properties: Optional[Dict[str, Any]] = None
```

Comment:
```diff
class ChatCompletionRequestCustomFields(ExtraForbidModel):
    configuration: Optional[Dict[str, Any]] = None
+   application_properties: Optional[Dict[str, Any]] = None
```

Reply:
moved. Pr in core: https://github.com/epam/ai-dial-core/pull/630